pkg/mdformatter/linktransformer/config.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Bartłomiej Płotka @bwplotka
+// Licensed under the Apache License 2.0.
+
+package linktransformer
+
+import (
+	"bytes"
+	"regexp"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v3"
+)
+
+type Config struct {
+	Version int
+
+	Validators []Validator `yaml:"validators"`
+}
+
+type Validator struct {
+	_regex  *regexp.Regexp
+	_maxNum int
+	// Regex for type github is reponame matcher, like `bwplotka\/mdox`.
+	Regex string `yaml:"regex"`
+	// By default type is `roundtrip`. Could be `github`.
+	Type string `yaml:"type"`
+}
+
+func ParseConfig(c []byte) (Config, error) {
+	if string(c) == "" {
```

Comment:
This can surprise. Imagine someone will pass empty bytes by mistake and parse will just succeed. Can we do this on caller side?

Reply:
Sure!